cibyl/plugins/openstack/deployment.py

```diff
@@ -42,14 +42,12 @@ class Deployment(Model):
             'attr_type': Node,
             'attribute_value_class': AttributeListValue,
             'arguments': [Argument(name='--nodes', arg_type=str,
-                          nargs='*',
```

Comment:
Why do we want to remove the `nargs='*'` option? Without it it would only parse one argument from --nodes

Reply:
ah.. sorry.. it got removed by mistake, I'll add it again.